app/backplane/power_module/include/c_power_module.h

```diff
@@ -45,8 +46,12 @@ class CPowerModule : public CProjectConfiguration {
 
 private:
     const char* ipAddrStr = (CREATE_IP_ADDR(NNetworkDefs::POWER_MODULE_IP_ADDR_BASE, 2, CONFIG_MODULE_ID)).c_str();
+    const char* sntpServerAddr = "10.2.1.1"; // TODO: Maybe we should look into hostnames? Also, still need to fix the create ip addr bug...
```

Comment:
If this is semi-permanent, wouldnt it be best to throw this in a conf?

Reply:
Is sntpServerAddr not just radio mod addr?